### DISCUSSION TITLE
Adding minification api to swc-loader

### DISCUSSION THREAD

**Discussion by floydjones1** [swc-project/swc#1993] [Q&A]
Hey I noticed now that MVP for minification is put on swc. I was trying to use it on swc-loader but I was banging my head against the table wondering why it wasn't getting minified. Turns out some changes need to happen in swc-loader. I made some naive changes in hopes that it would work but I didn't have any luck. Any estimate when that can be shipped? I can help if I understand it.

**Comment by kdy1** [comment]
I think fixing bugs of the minifier is more important but I think you can configure it via `.swcrc` or the option passed to the loader.

**Reply by floydjones1** [reply]
Yup totally understand, I see the issues coming in with minification. You got your hands tied. Can you at least confirm that some changes need to happen. Seems like all the loader allows to do right now is transform?

[https://github.com/swc-project/swc-loader/blob/master/src/index.js#L87](https://github.com/swc-project/swc-loader/blob/master/src/index.js#L87)

**Reply by floydjones1** [reply]
I'll make a PR as soon as I understand it. Don't worry 👍🏽

**Reply by kdy1** [reply]
Did you updated `@swc/core`? `swc.transform` should handle minification. There are some tests for it if I remember correctly.

**Reply by kdy1** [reply]
Oh, I think https://github.com/swc-project/swc/issues/1989 is relevant. I'll fix it with v.1.2.72

**Reply by floydjones1** [reply]
Ahh beautiful, thank you so much!
